bin/GameIndex.dbf

```diff
@@ -34770,6 +34770,12 @@ Region = NTSC-U
 Compat = 5
 eeClampMode = 3 // Objects appear in wrong places without it 
 vuClampMode = 2 // Fixes water reflection
+
```

Comment:
Extra row.

Reply:
It was intentional, the formatting matches similar cases and it's more readable.